content/doc/administration/requirements/jenkins-on-java-11.adoc

```diff
@@ -0,0 +1,66 @@
+---
+layout: documentation
+title: Running Jenkins on Java 11
+---
+
+If you need to upgrade the JVM version used to run Jenkins from Java 8 to Java 11, please link:/doc/administration/requirements/upgrade-java-guidelines[follow this guidelines].
+
+NOTE: At this moment, it is not required to upgrade the JVM version in order to run a Jenkins core more recent than `2.155`.
+
+Since Jenkins `2.155`, it is possible to run Jenkins with Java 11.
+However, it was greatly simplified since version `2.164`. 
+For the sake of simplicity, we will describe next how to run Jenkins on Java 11 with the most recent version of Jenkins.
+
+NOTE: We do encourage you to only use LTS versions for production purposes. 
+At the time of this writing, there is no LTS version with Java 11 Support.
+
+== Running Jenkins with Docker
+
+The easiest way to run Jenkins on Java 11 is to use the Docker image.
+Since the weekly release `2.155`, each weekly release has also been packaged on a Java 11 ready Docker image.
+All these images are based on the official link:https://hub.docker.com/r/_/openjdk/[openjdk:11-jdk] image maintained by the Docker Community.
+
+You can choose these Java 11 based images by using the `jdk11` tag (for the latest weekly release) or by suffixing the tag to use with `-jdk11` (eg. `2.163-jdk11`).
+
+This is the command to use, in order to start a Jenkins running on Java 11: 
+
+[source, shell]
+----
+docker pull jenkins/jenkins:jdk11
+docker run --rm -ti \
+  -p 8080:8080 -p 50000:50000
+  -v jenkins-home:/var/jenkins_home \
+  jenkins/jenkins:jdk11
+----
+
+== Running Jenkins with Java
+
+It is also possible to run Jenkins using the `java` command.
+
+However, we need to take some precaution with this method, because some plugins are using classes in the package `java.sql.*`.
+This package was modularized since Java 9, which means the classes inside this package are not accessible out-of-the-box.
+We need to open this module with the `java` parameter `--add-modules java.sql`.
+It is the case for example of plugin:pipeline-maven[Pipeline Maven integration] plugin but others are using classes from this package as well.
```

Comment:
I suppose you sure about that, but I thought this issue was not present anymore in "standard" Jenkins (i.e. not PCT, JFR, etc.), and you weren't able to reproduce this anymore.

Reply:
you are correct. I need to remove all this section.